docs/dev/test-configuration-options.md

```diff
@@ -557,37 +557,37 @@ Use this capability to enable animations for real devices by setting it to `true
+A `sauce:options` setting that controls Sauce Labs default resigning (iOS) or instrumentation (Android) of mobile apps installed on our devices. By default, this property is always `true`, but it can be set to `false` for private devices to allow testing of specific behaviors that are not permitted under the Sauce Labs provisioning. See [Resigning Enablements](/mobile-apps/automated-testing/ipa-files/#sauce-resigning-enablements) for more information.
 
 ---
 ### `sauceLabsImageInjectionEnabled`
 <p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> |</small></p>
 
-Appium override setting that enables the [camera image injection](/mobile-apps/features/camera-image-injection) feature.
+A `sauce:options` setting that enables the [camera image injection](/mobile-apps/features/camera-image-injection) feature.
 
 ---
 ### `sauceLabsBypassScreenshotRestriction`
 <p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> | <span className="sauceDBlue">Android Only</span> |</small></p>
 
-Appium override setting that bypasses the restriction on taking screenshots for secure screens (i.e., secure text entry).
+A `sauce:options` setting that bypasses the restriction on taking screenshots for secure screens (i.e., secure text entry).
 
 ---
 ### `allowTouchIdEnroll`
 <p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> |</small></p>
 
-Appium override setting that enables the interception of biometric input, allowing the test to simulate Touch ID interactions (not a Sauce Labs-specific capability).Z
+A `sauce:options` setting that enables the interception of biometric input, allowing the test to simulate Touch ID interactions (not a Sauce Labs-specific capability).Z
 
 ---
 ### `groupFolderRedirectEnabled`
 <p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span></small> | <small><span className="sauceDBlue">iOS Only</span> | </small></p>
 
-Appium override setting that enables the use of the app's private app container directory instead of the shared app group container directory. For testing on the Real Device Cloud, the app gets resigned, which is why the shared directory is not accessible.
+A `sauce:options` setting that enables the use of the app's private app container directory instead of the shared app group container directory. For testing on the Real Device Cloud, the app gets resigned, which is why the shared directory is not accessible.
 
 ---
 ### `systemAlertsDelayEnabled`
 <p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span></small> | <small><span className="sauceDBlue">iOS Only</span> | </small></p>
 
-Appium override setting that delays system alerts, such as alerts asking for permission to access the camera, to prevent app crashes at startup.
+A `sauce:options` setting that delays system alerts, such as alerts asking for permission to access the camera, to prevent app crashes at startup.
```

Comment:
We currently don't support the `sauce:options` on RDC, that will be supported when W3C is supported with will be around the 1st of December.

We will support both, the old JSONWP, which still be a capability on the level of the Appium capabilities, or with W3C with the `sauce:options`s

Reply:
Right. Hmmm. I expect we will need to rewrite this entire page as it relates to that. We have the "W3C" specific sections at the top, but they are only specific to Selenium. Once W3C is supported on Appium, those headings become confusing and I am guessing a lot of this content will need to be reworked. I have a ticket for that and will deal with it separately, but it is out of scope for this update, so what do you think is the best way to handle this for right now, @wswebcreation ?

Reply:
For now, I have removed any reference to configuration and just stated the capability purpose. We can deal with the syntax issues in my W3C updates branch. Thanks!